src/app/(account)/schemas.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from 'zod';
+
+export const emailSchema = z
+    .string()
+    .min(1, { message: 'Please enter your email' })
+    .email({ message: 'Please enter a valid email' });
+
+export const passwordSchema = z
+    .string()
+    .min(1, { message: 'Please enter a password' })
+    .min(8, { message: 'Password must be at least 8 characters' })
+    .regex(/^(?=.*[a-zA-Z]).+$/, { message: 'Password must include a letter' })
+    .regex(/^(?=.*[0-9]).+$/, { message: 'Password must include a number' });
+
+export const firstNameSchema = z
+    .string()
+    .min(1, { message: 'Please enter your first name' })
+    .regex(/^[a-zA-Z]+$/, {
+        message: 'Please enter a valid first name',
+    });
+
+export const lastNameSchema = z
+    .string()
+    .min(1, { message: 'Please enter your last name' })
+    .regex(/^[a-zA-Z]+$/, {
+        message: 'Please enter a valid last name',
+    });
```

Comment:
AFAIC it's too complex for supporting international names. And as a Chinese myself, supporting Chinese requires another library, so I think only English is fine.

Reply:
Even for English names, this regex is still too restrictive. It currently doesn't support hyphens or apostrophes which are not uncommon in English names either. And yes, while it may be harder to support HJK and therefore we shouldn't include support for it, I think we need to allow for more characters to be used.

Reply:
I agree that it should only support English characters due to complexity. Like Ray mentioned, hyphen and apostrophe support should be added.

Reply:
`/^[a-z ,.'-]+$/i` like this?